src/main.py

```
@@ -0,0 +1,163 @@
+
+st.set_page_config(page_title="ロジスティック方程式分析ツール", layout="wide")
+
+
+def extract_data_from_uploaded_file(
+    uploaded_file,
+) -> Optional[Tuple[np.ndarray, np.ndarray, str]]:
+    """アップロードされたExcelファイルからデータを抽出する"""
+    try:
+        df = pd.read_excel(uploaded_file, header=None)
+        if df.shape[1] < 2:
+            st.error("Excelファイルには少なくとも2つの列（時間と値）が必要です。")
```

Ruff (0.12.2)

String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)